modules/REST/V1/Languages.php

```diff
@@ -627,6 +627,28 @@ public function get_item_schema(): array {
 		return $this->add_additional_fields_schema( $this->schema );
 	}
 
+	/**
+	 * Retrieves an array of endpoint arguments from the item schema for the controller.
+	 * Ensures that the `no_default_cat` property is not returned for `EDITABLE` requests.
+	 *
+	 * @since 3.7
+	 *
+	 * @param string $method Optional. HTTP method of the request. The arguments for `CREATABLE` requests are
+	 *                       checked for required values and may fall-back to a given default, this is not done
+	 *                       on `EDITABLE` requests. Default WP_REST_Server::CREATABLE.
```

Comment:
Extra comments are relevant to the WordPress function:
```suggestion
	 * @param string $method Optional. HTTP method of the request. Default WP_REST_Server::CREATABLE.
```